app/templates/verify.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="ui icon info message eight wide column center aligned">
     <p>Your email has been verified successfully! Please login to continue.</p>
   </div>
-{{else}}
+{{else if (not success)}}
```

Comment:
if success if false, wouldn't `not success` be true by default? What is the significance of this?

Reply:
Else without a specified condition was printing until it was checking the status of "success", now it won't.

Reply:
okay, this seems like a bad implementation when it was first created, I will get back to you about this with a better fix, this shouldn't be happening in the first place.

Reply:
@CosmicCoder96 Any update on this?

Reply:
@prateekj117 Yes, go to the controller of the `verify` route. Introduce the missing `isLoading` state variable in it. Set it to true when the `verify` function is called. In the template display a loading state until `isLoading` is set to false in the `finally` clause of the request in `verify` function.